source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -388,7 +388,7 @@ export class CardDesignerSurface {
         this._cardHost.appendChild(renderedCard);
     }
 
-    private addPeer(peer: DesignerPeers.DesignerPeer) {
+    private addPeer(peer: DesignerPeers.DesignerPeer, neighbor: HTMLElement = undefined) {
```

Comment:
The code is hard to understand. Is it possible to use signature `addPeer(peer: DesignerPeers.DesignerPeer, bool insertAfterNeighbor)` and then calculate the neighbor during adding? (I don't know if it is easier or harder to read)

Reply:
Just updated and it's definitely easier to read - great suggestion! Let me know what you think